### PR TITLE
Add available options to their corresponding Wiki page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
Also moved the Bash code to a Python script

### CODE REVIEW

File: .github/scripts/checks-pages.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    html2text -b 0 "$f" > "$path"
+    add_footer "$path"
+
+    # Add entry to index
+    echo "" >> "$checks_index_path"
+    echo "[[$name|Check-$name]]" >> "$checks_index_path"
+done
+
+add_footer "$checks_index_path"
```
Comment (sebastiaanspeck) [inline .github/scripts/checks-pages.sh]:
```suggestion
add_footer "$checks_index_path"

```

**Comment by sebastiaanspeck** [comment]
For the result: https://github.com/sebastiaanspeck/magik-tools/wiki and https://github.com/sebastiaanspeck/magik-tools/wiki/Checks-Options.

By adding this to the Wiki, IMO https://github.com/StevenLooman/magik-tools/pull/135 will be superfluous. Maybe we can add a link in the `magik-lint`-help to create some awareness for the Wiki?

**Comment by StevenLooman** [comment]
I think I was too brief in #135. Why do the parameters need to be in on a separate page? Lets add the parameters to the `html` files of the checks. This way the parameters end up on the wiki automatically.

Adding a separate page adds something which has to be kept in sync manually. Yes, the html has to be kept in sync as well, but it is a bit "closer" to the implementation of the check itself.

**Comment by sebastiaanspeck** [comment]
> I think I was too brief in #135. Why do the parameters need to be in on a separate page? Lets add the parameters to the `html` files of the checks. This way the parameters end up on the wiki automatically.
> 
> 
> 
> Adding a separate page adds something which has to be kept in sync manually. Yes, the html has to be kept in sync as well, but it is a bit "closer" to the implementation of the check itself.

That makes things more clear, although adding the parameters to the `html` is maybe a bit redundant as in SonarQube they are also listed. I could also rewrite the Python file to extract the parameters from the *Check.java and add them to their corresponding Wiki Check-page. That way we do not have to update the HTML ourselves since it is copied straight from the *Check.java.

**Comment by StevenLooman** [comment]
Sounds like a less error prone way indeed. Please do so.

**Comment by sebastiaanspeck** [comment]
This somewhat depends on #140

**Comment by sebastiaanspeck** [comment]
Feel free to checkout my updated Wiki to see the result: 
https://github.com/sebastiaanspeck/magik-tools/wiki

- https://github.com/sebastiaanspeck/magik-tools/wiki/Checks-Index vs https://github.com/stevenlooman/magik-tools/wiki/Checks-Index (for the improved listing of the Checks)
- https://github.com/sebastiaanspeck/magik-tools/wiki/Check-FileMustStartWithPackageStatement vs https://github.com/stevenlooman/magik-tools/wiki/Check-FileMustStartWithPackageStatement (for the updated NOTE-footer).
- https://github.com/sebastiaanspeck/magik-tools/wiki/Check-Formatting vs https://github.com/stevenlooman/magik-tools/wiki/Check-Formatting (for the added Options-table, if applicable).

**Comment by StevenLooman** [comment]
Looks good, thanks!

**Pull request merged**